build.gradle

```diff
@@ -104,6 +104,7 @@ dependencies {
     implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.4'
     implementation group: 'com.microsoft.azure', name: 'applicationinsights-logging-logback', version: '2.6.4'
     implementation 'net.logstash.logback:logstash-logback-encoder:7.2'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
```

Comment:
I can't see where this dependency is used, can this be removed?

Reply:
removed